src/app/[locale]/topics/[topicId]/(topic)/store/features/flashcardSlice.ts

```
@@ -0,0 +1,48 @@
+            state.status = 'succeeded';
+            state.data = action.payload;
+        });
+        builder.addCase(fetchFlashcards.rejected, (state, action) => {
+            state.status = 'failed';
+            state.error = action.error.message ?? 'Unknown error';
+        });
+    },
+});
+
+export default flashcardsSlice.reducer;
+export const {} = flashcardsSlice.actions;
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)